src/util-server.ts

```diff
@@ -5,6 +5,7 @@
 
 // @ts-expect-error Package without types we can’t do anything about.
 import EleventyFetch from '@11ty/eleventy-fetch';
+import retry from 'p-retry';
 
 export type CachedFetchOptions = {
 	duration?: string;
```

Comment:
It would be great to have an optional `retryOptions?: Options` property here. `Options` is exported by `p-retry`. This should be passed to the `retry` call to allow configuring its behavior, including the number of retries and the function that gets called on error.

Reply:
I did consider this, but seeing as we don’t actually need it right now, I thought it probably falls under [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it)?

Reply:
True, we can always add it later if we actually need it.